features/auth/login/Login.tsx

```diff
@@ -9,40 +9,29 @@ import { useRouter } from "next/router";
 import { useEffect } from "react";
 import vercelLogo from "resources/vercel.svg";
 import { dashboardRoute, signupRoute } from "routes";
-import makeStyles from "utils/makeStyles";
 import stringOrFirstString from "utils/stringOrFirstString";
 
 import { useAuthContext } from "../AuthProvider";
 import useAuthStyles from "../useAuthStyles";
 import LoginForm from "./LoginForm";
 
-const useStyles = makeStyles((theme) => ({}));
-
 export default function Login() {
   const { t } = useTranslation([AUTH, GLOBAL]);
-  const { authState, authActions } = useAuthContext();
+  const { authState } = useAuthContext();
   const authenticated = authState.authenticated;
   const error = authState.error;
 
   const router = useRouter();
   const from = stringOrFirstString(router.query.from) ?? dashboardRoute;
   const redirectTo = from === "/" || from === "%2F" ? dashboardRoute : from;
-  const urlToken = stringOrFirstString(router.query.token);
 
   const authClasses = useAuthStyles();
-  const classes = useStyles();
 
   useEffect(() => {
     if (authenticated) {
       router.push(redirectTo);
     }
   }, [authenticated, router, redirectTo]);
```

Comment:
> how to handle redirects so that the user doesn't see the signup form when they are logged in (I'm a little confused about how this works now)

Would something like this hook not work? It might be worth extracting this into a custom hook if it's being used across `Login` and `Signup`